src/SAWScript/Interpreter.hs

```diff
@@ -1649,6 +1649,14 @@ primitives = Map.fromList
     , "given list of names, as defined with 'define', as uninterpreted."
     ]
 
+  , prim "w4_unint_z3_using" "String -> [String] -> ProofScript ()"
+    (pureVal w4_unint_z3_using)
+    Current
+    [ "Prove the current goal using What4 (Z3 backend) using the given"
+    , "Z3 tactic. Leave the given list of names, as defined with"
+    , "'define', as uninterpreted."
```

Comment:
What is `'define'` referring to in this context?

Reply:
copy-pasta from `w4_unint_z3`

Reply:
Oh, quite right! It'd be a shame to let this copy-pasta spread uncontrolled, however. My guess is that `define` is an anachronism, so perhaps we should just remove that bit. That is, simply state `Leave the given list of names as uninterpreted` (and similarly for `w4_unint_z3`).